docs/getting-started/auto-suggest.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: default
+title: Auto-Suggest
+nav_order: 5
+parent: Getting Started
+has_toc: false
+---
+
+# Intelligent Auto-Suggest
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+---
+
+## Introduction
+
+> Introduced in Tasks 1.9.0.
+
+The [Priorities]({{ site.baseurl }}{% link getting-started/priority.md %}), [Dates]({{ site.baseurl }}{% link getting-started/dates.md %}) and [Recurring Tasks]({{ site.baseurl }}{% link getting-started/recurring-tasks.md %}) pages show various emojis and special phrases that the Tasks plugin recognises, when searching for tasks.
+
+If you prefer to type your tasks, instead of using a dialog, there is now an intelligent auto-suggest completion mechanism that does a
+lot of the typing of emojis and dates for you.
+
+It is particularly powerful when creating and editing tasks on mobile phones.
+
+### Video Demo
+
+It is perhaps best understood by watching a [video of it in action](https://user-images.githubusercontent.com/10722656/175102574-78b0f851-cc48-4255-a40e-d3036bec5bb6.gif).
```

Comment:
Embedding this instead of linking (with `![](link.gif)`) might be nice here.

Reply:
I had thought the same. I started with it embedded, and as I was reading the docs, the movement was driving me nuts.

Also, I'm very conscious of this kind of effect:

https://www.includecpp.org/discord/#emojis-and-gifs

> For some people, animated emojis [and gigs] cause real, physical illness: it affects their brains badly. 

The rest of that text is about managing a Discord server, but I believe all the same factors apply to web pages too...